articles/quickstart/spa/react/00-intro.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Introduction
+description: An introduction to integrating Auth0 with your React applications
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - react
+  - login
+github:
+  path: 00-intro
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+<%= include('../_includes/_getting_started', { library: 'React', callback: 'http://localhost:3000/callback', returnTo: 'http://localhost:3000', showLogoutInfo: true, showWebOriginInfo: true, new_js_sdk: true }) %>
+
+<%= include('../_includes/_login_preamble', { library: 'React' }) %>
```

Comment:
I was going back and forth on whether to include this, and I did just for the first version mainly because it's there right now and I wasn't sure if we were able to _drop_ any info from what's currently on the tutorial. But it is at the very least in a weird place.

I also wasn't sure how relevant it is any more considering the new SDK hides a lot of this information for you. With the exception of the access token, the user isn't required to interact with the ID token or expiry timestamp at all, but this section of text makes it feel like the user should use those things in their applications.

Reply:
> I also wasn't sure how relevant it is any more considering the new SDK hides a lot of this information for you. With the exception of the access token, the user isn't required to interact with the ID token or expiry timestamp at all, but this section of text makes it feel like the user should use those things in their applications.

Yes that's true!  IMO, we should abstract the heavier bits of knowledge from the quickstart.  Treat the user on a need to know basis, and only inform the user when they are at a stage in the tutorial where they absolutely need this information.